pkg/didcomm/packer/jwe/authcrypt/authcrypt_test.go

```diff
@@ -569,15 +569,15 @@ func TestEncrypt(t *testing.T) {
 }
 
 func deepCopy(envelope, envelope2 *Envelope) {
-	for _, r := range envelope2.Recipients {
+	for i := 0; i < len(envelope2.Recipients); i++ {
```

Comment:
use range.

Reply:
rang is causing lint error

Reply:
what lint error?

Reply:
r is more than 128 bytes.. I reverted this by making r a pointer